### DISCUSSION TITLE
A key binding that enables full screen but the margin and bar is still visible

### DISCUSSION THREAD

**Discussion by GoodBoyNeon** [qtile/qtile#3924] [Q&A]
Currently, the full-screen function just makes it well... full-screen. But I wanna have some function that enables full-screen (so the rest of the windows ain't visible) but still want the bar and the margin too, if possible (the bar is a must!). How can I do that? I had that thing with xmonad using arcolinux's default config.

**Comment by m-col** [comment]
You could try using maximize instead of fullscreen, which doesn't cover any bars and positions a floating window. It won't respect gaps in the layout, as they are just that - part of the tiling layout. Thus if you want to have one window fill the space allocated to your layout, you can use the `Max` layout and toggle between that and your other layout(s) which will give you what you want.

**Reply by GoodBoyNeon** [reply]
> You could try using maximize instead of fullscreen, which doesn't cover any bars and positions a floating window. It won't respect gaps in the layout, as they are just that - part of the tiling layout. Thus if you want to have one window fill the space allocated to your layout, you can use the `Max` layout and toggle between that and your other layout(s) which will give you what you want.

Is there something that can maximize the current window, without switching to the max layout?  Something like `lazy.window.maximize`

**Reply by elParaguayo** [reply]
Close! `lazy.window.toggle_maximize()`.
